packages/core/src/domain/errorCollection.ts

```diff
@@ -6,25 +6,15 @@ import { Configuration, isIntakeRequest } from './configuration'
 import { monitor } from './internalMonitoring'
 import { computeStackTrace, Handler, report, StackFrame, StackTrace } from './tracekit'
 
-export interface ErrorMessage {
+export interface RawError {
   startTime: number
   message: string
-  context: {
-    error: ErrorContext
-    http?: HttpContext
-  }
-}
-
-export interface ErrorContext {
-  kind?: string
+  type?: string
   stack?: string
-  origin: ErrorSource
-}
-
-export interface HttpContext {
-  url: string
-  status_code: number
-  method: string
+  source: ErrorSource
+  url?: string
+  statusCode?: number
+  method?: string
 }
```

Comment:
I'm not sure to understand the motivation behind this change. `ErrorMessage` was convenient to assemble V1 events, now `RawError` is inconvenient for both V1 and V2 events. Also, `ErrorMessage` was a bit stricter on types (ex: the `HttpContext` made sure to be populated with all `url`, `status_code` and `method`).

Reply:
having an internal format for errors allow to have at some point the same structure for provided and auto error to mutualize the building of the v1 or v2 format between these errors. 
I flatten ErrorContext because properties were not consistently in this object and did not make sense in a product perspective. Keeping a structure for resource related properties could still make sens though.
I can go back on that.